src/modules/blocks/tickets/container/style.pcss

```diff
@@ -37,3 +37,39 @@
 		}
 	}
 }
+
+.tribe-editor__tickets{
+	.tickets-heading {
+		font-size: var( --tec-font-size-7 );
+		font-family: var( --tec-font-family-sans-serif );
+		font-style: normal;
+		font-weight: var( --tec-font-weight-regular );
+		line-height: var( --tec-line-height-1 );
+		margin-top: var( --tec-spacer-5 );
+		padding-bottom: var( --tec-spacer-1 );
+	}
+
+	.tickets-description {
+		color: #727272;
+		font-family: var( --tec-font-family-sans-serif );
+		font-size: var( --tec-font-size-3 );
+		font-style: normal;
+		font-weight: var( --tec-font-weight-regular );
+		line-height: var( --tec-line-height-3 );
+		padding: 16px 0;
+	}
+}
+
+.tribe-editor__action-dashboard.tribe-editor__tickets__action-dashboard.tribe-common {
+	background: #ffffff;
```

Comment:
```suggestion
	background: var(--tec-color-background);
```

Reply:
This was fixed using the tec color white var